### ISSUE TITLE
No devices found

### ISSUE THREAD

**Issue by anscarlett** [evilC/JoystickWrapper#11]
Hi,

Trying to get this to work, but it shows no devices (I have a Saitek X56 Throttle & Stick connected)

found what looks like a typo on line 160 of MonitorDemo.ahk

```this.Callback := callbacl```

any ideas?

**Comment by evilC** [comment]
Corrected typo, thanks.  
I take it the stick shows up fine in joy.cpl?  

If you debug on line 12 of monitor demo (`global DeviceList := jw.GetDevices()`), does DeviceList contain your devices?

**Comment by anscarlett** [comment]
Yes, it shows up fine. I've been testing your UCR AutoHotKey and it shows up in there fine as well.

i'll try the debugger when I get home in a few hours.

Thanks,

**Comment by anscarlett** [comment]
Ok, I tried debugging. I get no devices, the debugger skips straight over the loop. I also tried on a separate computer with a basic joystick connected with the same results.
